template/koa-rest-api/package.json

```diff
@@ -33,8 +33,8 @@
   },
   "devDependencies": {
     "@types/chance": "^1.1.3",
+    "@types/co-body": "^6.1.3",
```

Comment:
what's this

Reply:
To  un`skipLibCheck`, we need to pull in the transitive type dep 🥲

https://github.com/koajs/bodyparser/blob/031348a2d469dd288dc21cd10d8de280d9936f2c/package.json#L49

https://github.com/koajs/bodyparser/blob/031348a2d469dd288dc21cd10d8de280d9936f2c/src/body-parser.types.ts#L1

Reply:
😢 worth upstreaming?

Reply:
https://github.com/koajs/bodyparser/pull/164

Reply:
Thanks @samchungy 🙇